tests/Core/AllTests.php

```diff
@@ -46,6 +47,7 @@ public static function main()
     public static function suite()
     {
         $suite = new TestSuite('PHP CodeSniffer Core');
+        $suite->addTestSuite('PHP_CodeSniffer\Tests\Core\ConfigTest');
```

Comment:
I don't think this test is required - it can be removed. I would prefer any tests to be well thought out and submitted in a different PR (if you want to do that testing) rather than a quick adhoc test case. I really appreciate you doing it, but I don't think it is required for this change.

Reply:
Sorry, that's probably due to a remark I made earlier which was misunderstood.

I meant testing that the config is respected just for this sniff.

Reply:
It's useful now that you asked me to move the PHP_BINARY constant around.